omero/downloads/cli/help.out

```diff
@@ -1,141 +1,49 @@
-                ending. Lines containing zero or more keys along with a
-                single file to be imported. Options for formats include:
-                  - .tsv and .csv files will be parsed by the existing library
-                  - other files will be parsed with shlex
-                  - unless no columns are specified, in which case each line
-                    is treated as a file
-
-Optional Arguments:
-  In addition to any higher level options
-
-  -h, --help                            show this help message and exit
-  --javahelp, --java-help               Show the Java help text
-
-Login arguments:
-  Environment variables:
-
-    OMERO_USERDIR     Set the base directory containing the user's files.
-                      Default: $HOME/omero
-    OMERO_SESSIONDIR  Set the base directory containing local sessions.
-                      Default: $OMERO_USERDIR/sessions
-    OMERO_TMPDIR      Set the base directory containing temporary files.
-                      Default: $OMERO_USERDIR/tmp
-    OMERO_PASSWORD    Set the user's password for creating new sessions.
-                      Ignored if -w or --password is used.
-
-
-  Optional session arguments:
-
-  -C, --create                          Create a new session regardless of existing ones
-  -s SERVER, --server SERVER            OMERO server hostname
-  -p PORT, --port PORT                  OMERO server port
-  -g GROUP, --group GROUP               OMERO server default group
-  -u USER, --user USER                  OMERO username
-  -w PASSWORD, --password PASSWORD      OMERO password
-  -k KEY, --key KEY                     OMERO session key (UUID of an active session)
-  --sudo ADMINUSER                      Create session as this admin. Changes meaning of password!
-  -q, --quiet                           Quiet mode. Causes most warning and diagnostic messages to be suppressed.
-
-Python arguments:
-  Optional arguments which are used to configure import.
-
-  --bulk [BULK]                         Bulk YAML file for driving multiple imports
-  --logprefix [LOGPREFIX]               Directory or file prefix for --file and --errs
-  --file [FILE]                         File for storing the standard output from the Java process
-  --errs [ERRS]                         File for storing the standard error from the Java process
-  --clientdir CLIENTDIR                 Path to the directory containing the client JARs.  Default: lib/client
-  --logback LOGBACK                     Path to a logback xml file.  Default: etc/logback-cli.xml
-  --fetch-jars FETCH_JARS               Download this version of OMERO.java jars and exit
-  --depth DEPTH                         Number of directories to scan down for files
-  --skip {all,checksum,minmax,thumbnails,upgrade}
-                                        Optional step to skip during import
-  path                                  Path to be passed to the Java process
+Import any number of files into an OMERO instance.
```

Comment:
@jburel : this looks like import rather than download. Was that handled elsewhere?

Reply:
This is the output of 
```
omero import --javahelp 2> java-help.txt || echo "Dumped"
sed 1,5d java-help.txt > omero/downloads/cli/help.out
```
``help.out`` is then used in
```
omero/users/cli/import.rst:.. literalinclude:: /downloads/cli/help.out
```
The path name used at the time is not great

This has not been modified by the recent work